biz.aQute.repository/src/aQute/bnd/repository/p2/provider/P2Indexer.java

```diff
@@ -97,7 +98,7 @@ File get(String bsn, Version version, Map<String,String> properties, DownloadLis
 		if (link.isFile())
 			Files.delete(link.toPath());
```

Comment:
I think we also need to remove this delete since we want the createSymbolicLinkOrCopy method to be able to avoid the copy if the file was previously copied and there is no change to the source file.